File: src/codes/Users_Manual/segment/loop_behavior.rs
# use std::num::NonZeroU16;
# use autd3::prelude::*;
# fn main() {
# let _ = 
WithFiniteLoop {
    inner: Static::default(),
    loop_count: NonZeroU16::new(1).unwrap(),
    segment: Segment::S1,
    transition_mode: transition_mode::SyncIdx,
};
# }

File: src/codes/Users_Manual/segment/loop_behavior.cs
~using AUTD3Sharp;
~using AUTD3Sharp.TransitionMode;
~using AUTD3Sharp.Modulation;
new WithFiniteLoop(
    inner: new Static(),
    loopCount: 1,
    segment: Segment.S1,
    transitionMode: new SyncIdx()
);